mlrun/config.py

```diff
@@ -146,6 +146,8 @@
             "nosql": "v3io:///projects/{project}/fs/{kind}",
         },
         "default_targets": "parquet,nosql",
+        "default_job_image": "mlrun/mlrun",
+        "default_spark_image": "",
```

Comment:
We already have `spark_app_image` and `spark_app_image_tag` do we really need this ?

Reply:
done, left the job one since it may need extra packages over mlrun/mlrun